content/doc/book/installing/linux.adoc

```diff
@@ -162,8 +158,7 @@ sudo wget -O /etc/yum.repos.d/jenkins.repo \
     https://pkg.jenkins.io/redhat/jenkins.repo
 sudo rpm --import https://pkg.jenkins.io/redhat/jenkins.io.key
 sudo dnf upgrade
-sudo dnf install chkconfig java-devel
-sudo dnf install jenkins
+sudo dnf install java-devel jenkins
```

Comment:
Should this be more specifically defined as Java 11 so that future Fedora changes from the default Java 11 to Java 17 would not require a change to the instructions?

I'm not a Fedora experienced user.  I'm asking this as an open question, not as a specific request for a change.
```suggestion
sudo dnf install java-11-openjdk-devel jenkins
```

Reply:
In jenkinsci/packaging#275 I updated what the `jenkinsci/packaging` docs say to match what the `jenkinsci/packaging` tests do. In commit da40a72db and commit 33e77c68c I have updated what the `jenkins-infra/jenkins.io` docs say to match what the `jenkinsci/packaging` docs say and what the `jenkinsci/packaging` tests do. As a side effect of the above work, this pre-existring issue has been resolved.